src/types/data/index.sass

```diff
@@ -63,19 +64,13 @@
     padding-left: 17px
 
   &.date
-    display: flex
-    border: 1px solid #cccccc
-    border-radius: 4px
-
-    > input
-      height: 100%
-      padding: 10px
-      border-radius: 4px
-      font: inherit
+    position: relative
+    padding-left: 34px
 
-    > .react-date-field__calendar-icon
-      order: -1
-      margin: 2px 2px 0 4px
+    + .tether-target
```

Comment:
Plus is for sibling elements, correct? And since `tether-target` is the first child, this will never be selected.  I also think that this CSS is not needed since the tether library should handle this.